src/api/libcellml/units.h

```diff
@@ -417,47 +417,55 @@ class LIBCELLML_EXPORT Units: public NamedEntity, public ImportedEntity
      */
     size_t unitCount() const;
 
+    /**
+     * @brief Check whether there are any imported child @c Units.
+     *
+     * @return @c true when these @c Units rely on @c Units which are imported,
+     * or @c false otherwise.
+     */
+    bool requiresImports() const;
+
     /**
      * @brief Return the scaling factor difference between two @c Units.
      *
-     * This can be interpreted as factor, where units2 = (factor)*units1.  This method
-     * does not check to see if the units are compatible.
+     * This can be interpreted as factor, where units2 = (factor)*units1.  If the units
+     * are not compatible the factor returned is 0.0.
```

Comment:
Replace with:

```
* This can be interpreted as `factor`, where units2 = factor*units1.  If the units
* are not compatible the factor returned is 0.0.
```

? I.e. use `` ` `` around factor and remove the parentheses around its second occurrence.

Reply:
Done.